### ISSUE TITLE
Viewing Generated Schema

### ISSUE THREAD

**Issue by dbaggett** [google/rejoiner#18]
I have created my own example using the Library example as a reference. I have a graphql server, several gRPC services running and GraphiQL served up on 8080. It appears everything is in order based on the example.

However, I have not found anything that shows me what GraphQL queries are possible. Really, I would like to do introspection, but I know that is not supported by the underlying graphql-java library.

For example what would the following query be in GraphiQL:
`@Query("getAthletesByCountry")
    fun getAthletesByCountry(request: AthletesByCountryRequest, client: AthleteServiceGrpc.AthleteServiceFutureStub): ListenableFuture<Athletes> {
        return client.getAthletesByCountry(request)
    }`

Our ultimate goal is to have https://github.com/apollographql/apollo-android querying the server, but this would not be possible without a schema file.

I am probably missing something, so please forgive my ignorance.

**Comment by siderakis** [comment]
in GraphiQL you can press [control] + [space] to autocomplete.

For example to see the list of queries you can type `{` and then press [control] + [space]. You can also browse types in GraphiQL on the right side of the screen.

Based on the definition you provided above you can type `{ getAthletesByCountry { ... }}` and replace ... with the appropriate field selectors. 

The underlying library GraphQL-Java does provide introspection queries (at least with a JSON schema representation). Let me know if you'd like me to add that to an example.

**Comment by dbaggett** [comment]
I happened to stumble across an introspection [query example](https://gist.github.com/craigbeck/b90915d49fda19d5b2b17ead14dcd6da) that worked.

I was able to derive the correct query from the introspection response:
`query Athletes {
  getAthletesByCountry(input: {country: "Canada"}) {
    id
    first
    last
  }
}`

 The server returns a 500 - graphql.GraphQLException: no operation found - if an operation name is not provided which I am not sure is correct or not.

I would appreciate an introspection example still, if you don't mind.

On a sad note, I was not able to get autocomplete to work or types to display in GraphiQL.
